File: client/src/app/shared/models/subMenu.ts

export interface ISubMenu {
    id: number;
    menuHeadId: number;
    subMenuName: string;
    subMenuSeq: string;
    url: string;
}
 
export class SubMenu implements ISubMenu {
    id: number=0;
    menuHeadId: number;
    subMenuName: string;
    subMenuSeq: string;
    url: string;
} 

File: Core/Entities/SubMenu.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    public class SubMenu : BaseEntity
    {
        public int SubMenuSeq { get; set; }
        public string SubMenuName { get; set; }
        public string Url { get; set; }
        public int MenuHeadId { get; set; }

    }
}
